db/data/verifiers/invoice_default_external_view_key_verifier.rb

```
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class InvoiceDefaultExternalViewKeyVerifier
+  def self.verify!
+    Invoice.find_each do |invoice|
+      unless invoice.external_view_key?
+        raise "Invoice external key is present for Invoice ID #{invoice.id}"
```

The messaging here is weird, If the invoice.external_view_key is not present, we want to raise an error with something like "external_key is not populated for Invoice with id #{invoice.id}"